assets/app.js

```diff
@@ -105,10 +95,19 @@
             repo.language +
           '</p>'
         );
+        if ($('body').attr('data-prerendered') === 'data-prerendered') {
+          $link.find('.repo__info').append(
+            '<br>' +
+            'Updated ' +
+            prettyDate(repo.pushed_at)
```

Comment:
* All things now update on load (will update descriptions, watch count etc.) rather than rebuilding entire box each time
* updated element holds &amp;nbsp; in pre-render pass, gets updated on browser load.